index.php

```diff
@@ -533,6 +535,18 @@ public function createBackup() {
 		$this->silentLog('[info] end of createBackup()');
 	}
 
+	/**
+	* Returns a link to the changelog
+	* @return string
+	*
+	*/
+	private function getChangelogURL() {
+		$this->silentLog('[info] getChangelogURL()');
+		$changelogWebsite = 'https://nextcloud.com/changelog/';
+		$changelogURL = $changelogWebsite . '#' . str_replace('.', '-', substr($this->getConfigOption('version'),0,-2));
+		return $changelogURL;
+	}
+
```

Comment:
Could you add all those changes to the class also here to the file `Updater.php` to have parity in the CLI and web UI.

Reply:
Done!